### ISSUE TITLE
Error trying to install locally

### ISSUE THREAD

**Issue by antton** [play-with-go/play-with-go#391]
**Describe the bug**
I am trying to start the project and every time I run the docker-compose up I get the same output error.

**To Reproduce**
1. git clone the project
2. docker-compose up 
3.ERROR: Service 'controller' failed to build: the --mount option requires BuildKit. Refer to https://docs.docker.com/go/buildkit/ to learn how to build images with BuildKit enabled
4. I edit daemon.json adding 
"features": {
	       "buildkit": true
	},
5. After restart docker service
6. docker-compose up
7. ERROR: Service 'controller' failed to build: the --mount option requires BuildKit. Refer to https://docs.docker.com/go/buildkit/ to learn how to build images with BuildKit enabled

**Serverside (please complete the following information):**
 - OS:Ubuntu
 - Version: 20.04 LTS
 - Docker: 20.10.21
 - Docker-compose: 1.25.0

Any advice for the installation and/or build of the controller?

Thanks!!

**Comment by myitcv** [comment]
Looking at the error message:

```
the --mount option requires BuildKit. Refer to https://docs.docker.com/go/buildkit/ to learn how to build images with BuildKit enabled
```

It appears you don't have BuildKit (enabled). 

Does:

```
DOCKER_BUILDKIT=1 docker-compose up
```

work for you?

**Comment by antton** [comment]
Hi, 
Even if I did what you said I get the same error. Soo I had to change some dockerfiles to solve the build problems, but I still can't start the project at 100%. 

 Now the terminal of the guides show the following message `Initializing remote session` and in the docker logs gives the following error:

`controller_1 | failed to execute all presteps: failed to create prestep request for {http cmd_gitea:8080 /newuser false false }: Post "http://cmd_gitea:8080/newuser": EOF`


On the other hand I am unable to access the web panel from another computer through port 4000, but maybe we can postpone this to another issue or clarification.

Thanks in advance!
